---
title: "Kynon JM Benjamin"
image: "headshot_kjmb.jpg"
links:
  - label: LinkedIn
    url: "https://linkedin.com/in/kjbenjamin90"
  - label: Twitter
    url: "https://twitter.com/kjbenjamin90"
  - label: GitHub
    url: "https://github.com/Krotosbenjamin"
  - label: ORCiD
    url: "https://orcid.org/0000-0003-2016-4646"
  - label: Email
    url: "mailto:kj.benjamin90@gmail.com"
output:
  postcards::trestles
---

## Bio

Born and raised in a large extended family from Indianapolis, Indiana, Kynon Jade Benjamin is proud to be the first doctor in his family.
Dr. Benjamin earned his GED with the support of his mother before moving on to Indiana University–Purdue University Indianapolis (IUPUI).
At IUPUI, Dr. Benjamin completed his work study at a neuroscience research laboratory, which started his scientific research journey.
In his predoctoral studies, Dr. Benjamin designed and implemented drug delivery and drug development assays as well as developed
bioinformatic pipelines for differential expression analysis for Angelman syndrome - a neurodevelopmental disorder. In his subsequent
postdoctoral fellowship at the Lieber Institute for Brain Development and Johns Hopkins University School of Medicine, he developed
computational pipelines for large-scale transcriptional (bulk and single-cell), genetic, and functional associations analyses in
postmortem brain and brain (i.e., cerebral and striatal) organoids.

The **primary goal** of Dr. Benjamin's research is to improve therapeutics for under-research communities (i.e., personalized medicine)
via investigating the influence of genetic ancestry on neurological disorders in relevant tissues. To this end, he *uses and develops*
*computational tools* to examine the role of genetic ancestry in the brain. In addition to this, Dr. Benjamin has established
collaborations to use computational tools to address hypothesize driven question with publically available single-cell and bulk tissues.

Throughout his research career, Dr. Benjamin's experiences have reinforced the **critical need for diversity and creating inclusive**
**spaces**. As such, he has worked to provide mentorship and representation as well as advocate for opportunities for other
underrepresented minorities.

## Education
<div style="margin-left: auto;
            margin-right: auto;
            width: 100%">

| Institute / University                              | Location            | Degree/Field                                  | Completion Year |
|-----------------------------------------------------|---------------------|:---------------------------------------------:|:---------------:|
| Lieber Institute for Brain Development              | Baltimore, MD       | Postdoctoral Fellow in Computational Genetics | Current         |
| Johns Hopkins University School of Medicine         | Baltimore, MD       | Postdoctoral Fellow in Computational Genetics | Current         |
| Texas A&M University                                | College Station, TX | Ph.D. in Genetics                             | August 2017     |
| Rensselaer Polytechnic Institute                    | Troy, NY            | B.S. in Biomedical Engineering                | May 2012        |
| Indiana University, Purdue University, Indianapolis | Indianapolis, IN    | Biomedical Engineering (transferred)          | May 2010        |

</div> \

## Research Interests

#### Neurological research in African Americans to reduce health disparities (K99)

**Project**: Comprehensive Computational Analysis of Genetic and Regulatory Differences Between Individuals With
African and European Ancestries Across Four Brain Regions. Funding from National Institute on Minority Health and
Health Disparities ([K99 MD0169640](https://reporter.nih.gov/search/fZ-vbKVe5UCpRs4Cy1JD8Q/project-details/10351433)).

[MOSAIC](https://www.nigms.nih.gov/training/careerdev/Pages/MOSAIC.aspx) Scientific Society:
[The Association of American Medical Colleges](https://www.aamc.org/) (AAMC).\

In neuroscience and genomics, individuals with recent African ancestry (AA) account for less than 5% of large-scale
research cohorts for brain disorders but are 20% more likely to experience a major mental health crisis. Furthermore,
divergent responses to antipsychotics between AA and European ancestry (EA) have been linked to genetic differences.
Understanding these genetic and/or regulatory differences between AA and EA in the human brain, is essential to the
development of effective neurotherapeutics and potentially could decrease health disparities for neurological disorders.

1. Identify and characterize ancestry-associated expression differences in postmortem brain from AA and EA individuals.\
*Rationale*: Preliminary results revealed significant transcriptional changes across the four brain regions in neurotypical
controls with admixed AA associated with genetic ancestry. **Hypothesis**: Genetic variations influence ancestry-associated
transcriptional changes via alternative isoform usage. 1) Differently expressed (DE) genomic features (genes, transcripts,
exons, and junctions) associated with genetic ancestry will be identified and characterized within admixed AA. 2) eQTL
analysis will be preformed to identify genetic variants interacting with genetic ancestry and expression; and 3) The
contribution of these genetic variants on ancestry-associated expression differences will be determined.\
2. Identify and characterize of ancestry-associated epigenetic differences in the brain.\
*Rationale*: Transgenerational stress has been shown to affect health outcomes. Due to past policies and practices,
individuals of AA may be differentially exposed to extreme stresses across generations, which are identified as potential
risk factors for common disorders. **Hypothesis**: Epigenetic differences drive ancestry-associated expression differences in the
brain. 1) Differentially methylated regions (DMRs) and genetic variation associated with DNAm between ancestries will
be identified and characterized using BrainSeq Consortium publicly available expression, genetic, and DNA methylation
for the caudate nucleus (n=400) and dorsolateral prefrontal cortex (n=378). 2) The contribution of these DMRs and
DNAm on ancestry-associated expression differences will be determined.\
3. Evaluate ancestry-associated genetic and epigenetic correlations with complex traits in the postmortem brain.\
*Rationale*: The integration of genomic information with complex traits has been used to improve our understanding of
disease mechanism and prioritized potential therapeutic targets. Genetic differences have been linked to divergent
responses to antipsychotics, suggesting genetic background is important to understanding individual disease susceptibility.
**Hypothesis**: Epigenetic differences drive ancestry-specific complex trait associations in the brain. 1) Causal variants
impacting complex trains with ancestry-associated expression and DNAm will be identified and characterized. 2) Transfer
learning will be applied to improve causal variant detection for AA.

**Benjamin, KJM**, Chen, Q, Eagles, NJ, Huuki-Myers, LA, Collado-Torres, L, Stolz, JM, 
Shin, JH, Paquola, ACM, Hyde, TM, Kleinman, JE, Jaffe, AE, 
Han, S, and Daniel R Weinberger. ``Genetic and environmental contributions to ancestry differences in gene expression
in the human brain''. [preprint]. 2023.
DOI: [https://doi.org/10.21203/rs.3.rs-2922172/v1](https://www.researchsquare.com/article/rs-2922172/v1).

#### Software development

[dRFEtools](https://pypi.org/project/drfetools/): dynamic Recursive Feature Elimination\
Technology advances have generated larger ‘OMICs datasets with applications for machine learning. Even so, sample
availability results in smaller sample sizes compared to features. Dynamic recursive feature elimination (RFE) provides
a flexible feature elimination framework to tackle this problem. dRFEtools provides an interpretable and flexible tool
to gain biological insights from ‘OMICs data using machine learning.\
Authors: Kynon Jade Benjamin, Apuã Paquola, and Tarun Katipalli\
Accepted DOI: [https://doi.org/10.1093/bioinformatics/btad513](https://doi.org/10.1093/bioinformatics/btad513).

#### Collaborations

**Project**: Angiotensin II receptors in the human lung\
*Rationale*: The renin-angiotensin system is one of the most well characterized integrative pathways in humans and is known
to contribute to a variety of common disorders such as hypertension, chronic renal disease, and heart failure. The wide
availability of agents targeting this pathway has led to an expansion of its clinical spectrum to lung disorders. Despite
this widespread interest, specific localization of this receptor family in the vertebrate lung is limited. One reason is
due to the general imprecision of the available antibody reagents.\
**Goal**: Use publicly available single-cell and bulk RNA-sequencing to identify and characterize patterns of angiotensin II
receptor expression in the lung at different ages.

## Select Publications

**Benjamin, KJM**, Chen, Q, Jaffe, AE, Stolz, JM, Collado-Torres, L, Huuki-Myers, LA, Burke, EE, Arora, R, Feltrin, AS,
Barbosa, AR, Radulescu, E, Pergola, G, Shin, JH, Ulrich, WS, Deep-Soboslay, A, Tao, R, the BrainSeq Consortium,
Hyde, TM, Kleinman, JE, Erwin, JA, Weinberger, DR, and Apuã CM Paquola. ``Analysis of the caudate nucleus transcriptome
in individuals with schizophrenia highlights effects of antipsychotics and novel risk genes''. *Nature Neuroscience*.
2022. PMID: [36319771](https://www.nature.com/articles/s41593-022-01182-7).

**Benjamin, KJM<sup>+</sup>**, Arora, R<sup>+</sup>, D'Ignazio, L, Hyde, TM, Kleinman, JE, Weinberger, DR, Paquola, ACM,
and Jennifer A Erwin. ``Transcriptional and genetic sex differences for schizophrenia across the dorsolateral
prefrontal cortex, hippocampus, and caudate nucleus''. [preprint]. 2022.
DOI: [https://doi.org/10.1101/2022.09.30.22280452](https://doi.org/10.1101/2022.09.30.22280452).

Sawada, T, **Benjamin, KJM**, Brandtjen, AC, Tietze, E, Allen, SJ, Paquola, ACM, Kleinman, JE, Hyde, TM, and Jennifer A
Erwin. ``Generation of four postmortem dura-derived iPS cell lines from four control individuals with genotypic and
brain-region-specific transcrptomic data available through the BrainSEQ consortium''. *Stem Cell Research*. 2020.
PMID: [32446240](https://doi.org/10.1016/j.scr.2020.101806).

## Past Funding

Grant: T32 MH015330, 06/01/2019-05/31/2021\
Funding Source: NIH/NIMH\
Role: Postdoctoral Fellow\
This Interdisciplinary Training Program in Psychiatry and Neuroscience provides postdoctoral research training in areas
relevant to the neurobiological bases of mental disorders. The close interaction with the Lieber Institute provides
additional training opportunities with large-scale postmortem genetic and regulatory studies.

Grant: CVM Advanced Developmental Training Travel Award, 05/01/2015-08/31/2015\
Funding Source: Texas A&M University College of Veterinary Medicine & Biomedical Sciences\
Role: PI\
This is a travel award to attend the Cold Spring Harbor Laboratory *Drosophila* Neurobiology Course ($2500).

Grant: Great Lakes National STEM Scholarship, 07/01/2014\
Funding Source: Great Lakes\
Role: PI\
This is a scholarship program for individuals in the STEM (science, technology, engineering, and mathematics) open to
high school students, college students, and graduate students ($2500).

Grant: CVM Graduate Student Research Trainee Grant, 05/01/2014\
Funding Source: Texas A&M University College of Veterinary Medicine & Biomedical Sciences\
Role: PI\
This is a graduate student research trainee grant for research and supplies to generate preliminary data ($5000).

Grant: WSGI Graduate Traineeship, 09/01/2013-08/31/2014\
Funding Source: Texas A&M Institute for Genome Sciences and Society (WSGI)\
Role: PI\
This is a competitive graduate traineeship that supports half of salary for computational analysis of 50 idiopathic
Angelman Syndrome patients.

## Additional Information

[CV](./resume_cv.pdf)\
[NCBI Bibliography](https://www.ncbi.nlm.nih.gov/myncbi/kynon%20jade.benjamin.1/bibliography/public/)
