---
title: "Kynon JM Benjamin"
image: "headshot_kjmb.jpg"
links:
  - label: LinkedIn
    url: "https://linkedin.com/in/kjade-benjamin13"
  - label: Twitter
    url: "https://twitter.com/kjbenjamin90"
  - label: GitHub
    url: "https://github.com/Krotosbenjamin"
  - label: Email
    url: "mailto:kj.benjamin90@gmail.com"
output:
  postcards::trestles
---

## Bio

Born and raised in a large extended family from Indianapolis, Indiana, Kynon Jade Benjamin is proud to be the first doctor in his family.
Dr. Benjamin earned his GED with the support of his family before moving on to Indiana University–Purdue University Indianapolis (IUPUI).
At IUPUI, Dr. Benjamin completed his work study at a neuroscience research laboratory, which started his scientific research journey.
In his predoctoral studies, Dr. Benjamin designed and implemented drug delivery and drug development assays as well as developed
bioinformatic pipelines for differential expression analysis for Angelman syndrome - a neurodevelopmental disorder. In his subsequent
postdoctoral fellowship at the Lieber Institute for Brain Development and Johns Hopkins University School of Medicine, he developed
computational pipelines for large-scale transcriptional (bulk and single-cell), genetic, and functional associations analyses in
postmortem brain and brain (cerebral and striatal) organoids.

The **primary goal** of Dr. Benjamin's research is to improve therapeutics for under-research communities (i.e., personalized medicine)
via investigating ancestry differences for neurological disorders in relevant tissues. To this end, he *uses and develops computational*
*tools* to examine the role of ancestry in the brain. In addition to this, Dr. Benjamin has established collaborations to use
computational tools to address hypthesize driven question with publically available single-cell and bulk tissues.

Throughout his research career, Dr. Benjamin's experiences have reinforced the **critical need for diversity and creating inclusive**
**spaces**. As such, he has worked to provide mentorship and representation as well as advocate for opportunities for other
underrepresented minorities.

## Education
<div style="margin-left: auto;
            margin-right: auto;
            width: 100%">

| Institute / University                              | Location            | Degree/Field                                  | Completion Year |
|-----------------------------------------------------|---------------------|:---------------------------------------------:|:---------------:|
| Lieber Institute for Brain Development              | Baltimore, MD       | Postdoctoral Fellow in Computational Genetics | Current         |
| Johns Hopkins University School of Medicine         | Baltimore, MD       | Postdoctoral Fellow in Computational Genetics | Current         |
| Texas A&M University                                | College Station, TX | Ph.D. in Genetics                             | August 2017     |
| Rensselaer Polytechnic Institute                    | Troy, NY            | B.S. in Biomedical Engineering                | May 2012        |
| Indiana University, Purdue University, Indianapolis | Indianapolis, IN    | Biomedical Engineering (transferred)          | May 2010        |

</div> \


## Research Interests

#### Neurological research in African Americans to reduce health disparities (K99)

**Project Title**:            Comprehensive Computational Analysis of Genetic and Regulatory Differences Between Individuals With African and European Ancestries Across Four Brain Regions\
Funding NIH Institute/Center: National Institute on Minority Health and Health Disparities\
Grant ID:                     [K99MD0169640](https://reporter.nih.gov/search/fZ-vbKVe5UCpRs4Cy1JD8Q/project-details/10351433)\
[MOSAIC](https://www.nigms.nih.gov/training/careerdev/Pages/MOSAIC.aspx) Scientific Society: [The Association of American Medical Colleges](https://www.aamc.org/) (AAMC)\


In neuroscience and genomics, individuals with recent African ancestry (AA) account for less than 5% of large-scale
research cohorts for brain disorders but are 20% more likely to experience a major mental health crisis. Furthermore,
divergent responses to antipsychotics between AA and European ancestry (EA) have been linked to genetic differences.
Understanding these genetic and/or regulatory differences between AA and EA in the human brain, is essential to the
development of effective neurotherapeutics and potentially could decrease health disparities for neurological disorders.

1. Identify and characterize ancestry-related expression differences in postmortem brain from AA and EA individuals.\
*Rationale*: Preliminary results revealed significant transcriptional changes across the four brain regions in neurotypical
controls with admixed AA associated with genetic ancestry. **Hypothesis**: Genetic variations influence ancestry-related
transcriptional changes via alternative isoform usage. 1) Differently expressed (DE) genomic features (genes, transcripts,
exons, and junctions) associated with genetic ancestry will be identified and characterized within admixed AA. 2) eQTL
analysis will be preformed to identify genetic variants interacting with genetic ancestry and expression; and 3) The
contribution of these genetic variants on ancestry-related expression differences will be determined.\
2. Identify and characterize of ancestry-related epigenetic differences in the brain.\
*Rationale*: Transgenerational stress has been shown to affect health outcomes. Due to past policies and practices,
individuals of AA may be differentially exposed to extreme stresses across generations, which are identified as potential
risk factors for common disorders. **Hypothesis**: Epigenetic differences drive ancestry-related expression differences in the
brain. 1) Differentially methylated regions (DMRs) and genetic variation associated with DNAm between ancestries will
be identified and characterized using BrainSeq Consortium publicly available expression, genetic, and DNA methylation
for the caudate nucleus (n=400) and dorsolateral prefrontal cortex (n=378). 2) The contribution of these DMRs and
DNAm on ancestry-related expression differences will be determined.\
3. Evaluate ancestry-related genetic and epigenetic correlations with complex traits in the post-mortem brain.\
*Rationale*: The integration of genomic information with complex traits has been used to improve our understanding of
disease mechanism and prioritized potential therapeutic targets. Genetic differences have been linked to divergent
responses to antipsychotics, suggesting genetic background is important to understanding individual disease susceptibility.
**Hypothesis**: Epigenetic differences drive ancestry-specific complex trait associations in the brain. 1) Causal variants
impacting complex trains with ancestry-related expression and DNAm will be identified and characterized. 2) Transfer
learning will be applied to improve causal variant detection for AA.

#### Software development

[dRFEtools](https://pypi.org/project/drfetools/): dynamic Recursive Feature Elimination\
Technology advances have generated larger ‘OMICs datasets with applications for machine learning. Even so, sample
availability results in smaller sample sizes compared to features. Dynamic recursive feature elimination (RFE) provides
a flexible feature elimination framework to tackle this problem. dRFEtools provides an interpretable and flexible tool
to gain biological insights from ‘OMICs data using machine learning.\
Authors: Kynon Jade Benjamin, Apuã Paquola, and Tarun Katipalli

#### Collaborations


## Select Publications

